pkg/oc/cli/debug/debug.go

```diff
@@ -562,6 +554,7 @@ func (o *DebugOptions) getContainerImageCommand(pod *kapi.Pod, container *kapi.C
 		image, _ = o.getContainerImageViaImageStreamImport(container)
 	}
 
+	// TODO: DockerImageMetadata is a RawExtension in the external api - how to access config?
 	if image == nil || image.DockerImageMetadata.Config == nil {
```

Comment:
@deads2k @soltysh Not sure how to access config here - DockerImageMetadata is a RawExtension in the external api